docs/docs/api.md

```diff
@@ -1042,6 +1041,18 @@ Parameters:
 
 Requires no auth.
 
+## Internal API
+
+Manifold has some internal API endpoints that are not part of the official API. These are largely undocumented, but a few are mentioned here for third-party use until a more permanent solution is implimented. These endpoints are not preceeded by `/v0` and are even more subject to sudden changes than the official API endpoints.
+
+### `POST /unresolve`
+
+Unresolves a market. Requires auth.
+
+Parameters:
+
+- `contractId`: The ID of the market to unresolve.
+
```

Comment:
Not really jazzed about including this. There's a 0% chance anyone will ever update this if the `unresolve` endpoint changes, and if moderators want this endpoint, we should put it in the `v0` namespace, not tell them to use this one.

Reply:
Wanna bet against me at 0%? :) Or even 10%? The fact that I've added it to the documentation is strong evidence against your claim.

In general, yes, of course it would be better for this to be a more official, reliable endpoint. Feel free to add a v0 version. But right now there is no such official unresolve endpoint, so it's better for people to be aware of the one that does exist. "Either we document this or we make it better" is a false dichotomy; what's likely to actually happen is that it will be neither fixed nor documented. For the time being while it's not fixed, being documented seems better than not being documented.